axelrod/tests/strategies/test_titfortat.py

```diff
@@ -678,7 +678,7 @@ def test_output_from_literature(self):
         expected_scores = [
             3000.0,
             915.0,
-            2763.0,
```

Comment:
What's happening here? I believe these expected scores are actually from the literature?

Reply:
Ignore that, read closer and see that it's just the random opponent (I panicked 😆 ).

Reply:
Oh good catch, I think I got this right in the documentation test. Let me try to reconcile.